### PR TITLE
add property colors

### PR DESCRIPTION
**Pull request by FilipeQ**
add property color and docs

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -50,6 +50,13 @@ Select an optimization level between `1` and `3`.
 2. Also uses transparency to shrink the file further.
 3. Try several optimization methods (usually slower, sometimes better results)
 
+##### colors
+
+Type: `number`<br>
+Default: `undefined`
```
Comment (kevva) [inline readme.md]: No need for a default if there is none.

Comment (FilipeQ) [inline readme.md]: yes :)

**Comment by shinnn** [comment]
@FilipeQ Can this option actually solve https://github.com/imagemin/imagemin-gifsicle/issues/11?

File: readme.md [hunk 2]
```diff
@@ -50,6 +50,12 @@ Select an optimization level between `1` and `3`.
 2. Also uses transparency to shrink the file further.
 3. Try several optimization methods (usually slower, sometimes better results)
 
+##### colors
+
+Type: `number`<br>
```
Comment (shinnn) [inline readme.md]: `<br>` is not needed here.

**Comment by FilipeQ** [comment]
yes this solve the problem.

**Comment by kevva** [comment]
Fixed in https://github.com/imagemin/imagemin-gifsicle/commit/c85ead9103f98c46b65f765d42a7dcb59d56dea1.

**Pull request closed without merging**